app/src/main/java/com/example/campusguide/MapsActivity.kt

```diff
@@ -60,6 +62,24 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
         val hall = LatLng(45.497290, -73.578824)
         mMap.addMarker(MarkerOptions().position(hall).title("Hall Building"))
         mMap.animateCamera(CameraUpdateFactory.newLatLngZoom(hall, 17.0f))
+
+        // Update switch campus button listener
+        val switchCampusToggle: ToggleButton = findViewById(R.id.switchCampusButton)
+        switchCampusToggle.setOnCheckedChangeListener{ _, isChecked ->
+            switchCampus(isChecked)
+        }
+    }
+    
+    private fun switchCampus(isChecked: Boolean) {
+        if(isChecked){
+            // isChecked = true = on then we are at sgw campus, switch to loyola campus
+            val loyCoord = LatLng(45.458153, -73.640490)
+            mMap.animateCamera(CameraUpdateFactory.newLatLngZoom(loyCoord, 17.0f))
```

Comment:
We've used 17.0f several times in the class, it's probably time to extract it into a constant variable so we can easily adjust our zoom level

Reply:
i'm not sure how appropriate it is to save 17.0f as a string constant, so would a new xml file in values be better? would i just call it "constants"?